flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/collect/CollectSinkFunction.java

```diff
@@ -383,12 +383,10 @@ public void run() {
 
 					if (!version.equals(requestVersion) || requestOffset < offset) {
 						// invalid request
-						if (LOG.isDebugEnabled()) {
-							// this is normal for the 1st request after the sink (re)starts, so we print debug log
-							LOG.debug("Invalid request. Received version = " + requestVersion +
-								", offset = " + requestOffset + ", while expected version = "
-								+ version + ", offset = " + offset);
-						}
+						// this is normal for the 1st request after the sink (re)starts
+						LOG.info("Invalid request. Received version = " + requestVersion +
```

Comment:
How many these logs will there be? Will it flow the log file?

Reply:
Use `LOG.warn`?

Reply:
It won't flow the log file. This log only happens when the sink starts or restarts.

This is a mechanism to inform the collect iterator that the sink has restarted and is an expected behavior. So an info log is enough.

Reply:
Thanks for the explanation.